File: Official Windows Platform Sample/Windows Phone 8.1 samples/[JavaScript]-Windows Phone 8.1 samples/Display orientation sample/JavaScript/WindowsPhone/js/AppUIConstants.js
//// THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF
//// ANY KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO
//// THE IMPLIED WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A
//// PARTICULAR PURPOSE.
////
//// Copyright (c) Microsoft Corporation. All rights reserved

(function () {
    "use strict";

    // Constants specific to the Windows Phone version.
    WinJS.Namespace.define("Constants", {
        // On Windows the accelerometer returns values relative to Portrait orientation.
        // This sample always treats rotation as relative to Landscape regardless of platform.
        "UIAngleOffset": 180
    });
})();

File: Official Windows Platform Sample/Universal Windows app samples/111487-Universal Windows app samples/Display orientation sample/C#/WindowsPhone/AppUIConstants.cs
//*********************************************************
//
// Copyright (c) Microsoft. All rights reserved.
// THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
// ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
// IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
// PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
//
//*********************************************************

namespace SDKTemplate
{
    /// <summary>
    /// Constants specific to the Windows Phone version.
    /// </summary>
    static class Constants
    {
        // On Phone the accelerometer returns values relative to Portrait orientation.
        // This sample always treats rotation as relative to Landscape regardless of platform.
        public const int UIAngleOffset = 180;
    }
}